django-stubs/views/generic/edit.pyi

```diff
@@ -66,9 +66,9 @@ class DeletionMixin(Generic[_M]):
     def delete(self, request: HttpRequest, *args: Any, **kwargs: Any) -> HttpResponse: ...
     def get_success_url(self) -> str: ...
 
-class BaseDeleteView(DeletionMixin[_M], BaseDetailView[_M]):
+class BaseDeleteView(DeletionMixin[_M], FormMixin[_ModelFormT], BaseDetailView[_M]):
```

Comment:
It is better to explicitly add `Generic[_M, _ModelFormT]` when two vars are used this way.

Otherwise, it is easy to mix things up.

Reply:
So, like the commit I just pushed?

Reply:
Yes :)